app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
```

Comment:
- [ ] Dear Team, Your project is great but I could not confirm your `layouts` since I fail to run the application locally. Currently when I run `db:setup` or `db:prepare` I face an error with the seeder file. 
![Screen Shot 2023-09-08 at 11 14 53 AM](https://github.com/emmanuelsteven/MyBlogApp/assets/69077061/9ad37c0d-cc73-4c20-bc07-d57fe81938d9)
**And**
- [ ] When I Ignore the seeder fine and run the setup with `db:drop db:create db:migrate`, the application starts, but I still face an issue with the device. 
![Screen Shot 2023-09-08 at 11 18 35 AM](https://github.com/emmanuelsteven/MyBlogApp/assets/69077061/a4e6e1f3-bf78-41c3-90dd-fd5d590418d4)
![Screen Shot 2023-09-08 at 11 18 49 AM](https://github.com/emmanuelsteven/MyBlogApp/assets/69077061/648b875d-fb93-43f1-83b7-a4d46ea5fd1c)
**Please make sure the application can run without error and resubmit for a review.**

- [Optional] Please remember, when ever we add content to the `seeder` file we need to make sure its valid and can be used everywhere. If not the case, its better to leave it empty.